src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -126,7 +186,8 @@ private void createTopic(final String topic, final int partitionsNum, final int
                 final Properties topicConfig = new Properties();
                 topicConfig.setProperty("retention.ms", Long.toString(retentionMs));
                 topicConfig.setProperty("segment.ms", Long.toString(rotationMs));
-                AdminUtils.createTopic(zkUtils, topic, partitionsNum, replicaFactor, topicConfig);
+                AdminUtils.createTopic(zkUtils, topic, partitionsNum, replicaFactor, topicConfig,
+                        RackAwareMode.Enforced$.MODULE$);
```

Comment:
I'm reading this part of the client code https://github.com/apache/kafka/blob/cae5977ed0d6b63f992973800273769c970b0a0a/core/src/main/scala/kafka/admin/RackAwareMode.scala#L22-L39 and trying to understand the change in behaviour with `RackAwareMode.Enforced`.

Do you understand the difference between `Safe`, `Disabled` and `Enforced`? It's not clear for me.